[BUG] AI suggestions fail when local LLM omits required empty array fields

### Description

### Description

When using the AI suggestions feature with an OpenAI-compatible local LLM, Paperless-ngx fails if the model omits one of the required array fields from the tool-call arguments.

In my case, the model returned a valid tool call with most fields populated, but omitted the `dates` field because no suitable date was identified.

Paperless then raises a Pydantic validation error instead of treating the missing array as an empty list.


#### Expected behavior
If an LLM omits an array field because no matching value was found, Paperless should treat the field as an empty list.
For example:
{
  "dates": []
}
The AI suggestion request should not fail completely just because one optional classification category was omitted from the tool-call arguments.

#### Actual behavior
The complete AI suggestion request fails with a Pydantic validation error.
No suggestions are displayed, even though the model returned valid values for fields such as:
- title
- correspondent
- tags
- document type
- storage path

### Paperless-ngx version

```text
ghcr.io/paperless-ngx/paperless-ngx:3.0

### Steps to reproduce

Backend: openai-like
Inference server: llama.cpp
Model: Qwen2.5-1.5B-Instruct Q8_0

[ERROR] [paperless.api] Invalid AI configuration while generating suggestions for document 429: 1 validation error for DocumentClassifierSchema

dates
  Field required [type=missing, input_value={'title': 'Paper...'}, input_type=dict]

### Webserver logs

```bash
Traceback (most recent call last):
  File "/usr/src/paperless/src/documents/views.py", line 1543, in ai_suggestions
    llm_suggestions = get_ai_document_classification(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/paperless/src/paperless_ai/ai_classifier.py", line 156, in get_ai_document_classification
    localized = client.run_llm_query(
                ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/paperless/src/paperless_ai/client.py", line 149, in run_llm_query
    parsed = DocumentClassifierSchema(**tool_calls[0].tool_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pydantic/main.py", line 250, in __init__
    validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

pydantic_core._pydantic_core.ValidationError: 1 validation error for DocumentClassifierSchema

dates
  Field required [type=missing, input_value={'title': 'Paper...'}, input_type=dict]
```

### Browser logs

```bash

```

### Services logs

```bash

```

### Paperless-ngx version

3.0

### Host OS

Ubuntu 24

### Installation method

Docker - official image

### System status

```json

```

### Browser

_No response_

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

--- a/src/paperless_ai/base_model.py
+++ b/src/paperless_ai/base_model.py
@@ -1,12 +1,13 @@
 from pydantic import BaseModel
+from pydantic import Field
 
 
 class DocumentClassifierSchema(BaseModel):
     """Schema for document classification suggestions."""
 
     title: str
-    tags: list[str]
-    correspondents: list[str]
-    document_types: list[str]
-    storage_paths: list[str]
-    dates: list[str]
+    tags: list[str] = Field(default_factory=list)
+    correspondents: list[str] = Field(default_factory=list)
+    document_types: list[str] = Field(default_factory=list)
+    storage_paths: list[str] = Field(default_factory=list)
+    dates: list[str] = Field(default_factory=list)

--- a/src/paperless_ai/tests/test_base_model.py
+++ b/src/paperless_ai/tests/test_base_model.py
@@ -1,0 +1,35 @@
+import pytest
+from pydantic import ValidationError
+
+from paperless_ai.base_model import DocumentClassifierSchema
+
+
+@pytest.mark.parametrize(
+    "omitted_field",
+    [
+        "tags",
+        "correspondents",
+        "document_types",
+        "storage_paths",
+        "dates",
+    ],
+)
+def test_document_classifier_schema_defaults_omitted_list_field(omitted_field):
+    data = {
+        "title": "Test Title",
+        "tags": ["test"],
+        "correspondents": ["Test Correspondent"],
+        "document_types": ["Test Document Type"],
+        "storage_paths": ["Test Storage Path"],
+        "dates": ["2026-07-31"],
+    }
+    del data[omitted_field]
+
+    result = DocumentClassifierSchema(**data)
+
+    assert getattr(result, omitted_field) == []
+
+
+def test_document_classifier_schema_requires_title():
+    with pytest.raises(ValidationError, match="title"):
+        DocumentClassifierSchema()

